unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/Services/RestrictedActionsService/RestrictedActionsServiceImpl.cs

```diff
@@ -86,11 +86,13 @@ private static int GetCurrentFrameCount()
             return Time.frameCount;
         }
 
-        // TODO: change this when support for wearables/emotes/etc urn is added
-        // urn format urn:<CHAIN>:<CONTRACT_STANDARD>:<CONTRACT>:<TOKEN_ID> i.e: urn:ethereum:erc721:0x00...000:123
-        internal static bool TryParseUrn(string urn, out string contractAddress, out string tokenId)
+        // TODO: update this when support for wearables/emotes/etc urn is added
+        // urn format urn:decentraland:<CHAIN>:<CONTRACT_STANDARD>:<CONTRACT_ADDRESS>:<TOKEN_ID>
+        // i.e: urn:decentraland:ethereum:erc721:0x00...000:123
+        public static bool TryParseUrn(string urn, out string contractAddress, out string tokenId)
```

Comment:
could we move this to another class/helper so we can use it here and elsewhere without having to include some rpc service asmdef anywhere else?

Reply:
Sure, I'll move it to NFTUtils then